pkg/cmd/cli/cmd/project.go

```diff
@@ -66,7 +66,6 @@ func NewCmdProject(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.
 		Short:   "Switch to another project",
 		Long:    projectLong,
 		Example: fmt.Sprintf(projectExample, fullName),
-		Aliases: []string{"projects"},
```

Comment:
I'm ok with this being a breaking change given the gains. But I'd like to minimize the breakage by making the exact current message still present in the output message of both `project` and `projects`.

```
$ oc project
Using project "awesome" on server "https://127.0.0.1:8443".

$ oc projects
You have access to the following projects and can switch between them with 'oc project <projectname>':

  * awesome (current)
  * notsoawesome
  * okone
  * weirdo

Using project "awesome" on server "https://127.0.0.1:8443".
```

That's also consistent with how we print the projects after logging in, and printing the current one at the end should address the use cases of having a long list of projects. 

Comments?